NitroxPatcher/Patches/Dynamic/Bench_OnHandClick_Patch.cs

```diff
@@ -33,8 +33,8 @@ public static bool Prefix(Bench __instance, GUIHand hand)
                 return true;
             }
 
-            HandInteraction<Bench> context = new HandInteraction<Bench>(__instance, hand);
-            LockRequest<HandInteraction<Bench>> lockRequest = new LockRequest<HandInteraction<Bench>>(id, SimulationLockType.EXCLUSIVE, ReceivedSimulationLockResponse, context);
+            HandInteraction<Bench> context = new(__instance, hand);
+            LockRequest<HandInteraction<Bench>> lockRequest = new(id, SimulationLockType.EXCLUSIVE, ReceivedSimulationLockResponse, context);
```

Comment:
will this use the bench segment id or the bench id?

Reply:
There is no `Bench` component on the root anymore, just on the segments which are used for the lock.